DMCompiler/Compiler/DM/DMParser.cs

```diff
@@ -1159,6 +1159,22 @@ private DMASTProcStatementSpawn Spawn() {
             return new DMASTProcStatementSpawn(loc, delay ?? new DMASTConstantInteger(loc, 0), body);
         }
 
+        private void ExtraColonPeriod() {
+            var token = Current();
+            if (token.Type is not (TokenType.DM_Colon or TokenType.DM_Period))
+                return;
+
+            Advance();
+
+            if (Current().Type is not (TokenType.DM_Semicolon or TokenType.Newline) && !WhitespaceTypes.Contains(Current().Type))
```

Comment:
Would `Delimiter()` work better here than `Current().Type is not (TokenType.DM_Semicolon or TokenType.Newline)`? That matches multiple semicolons/newlines if there are any.